On `ros2` compile requires Ament tricks

## Steps to Reproduce the Problem

  1. Install ROS 2 dashing from ubuntu packages
  1. Clone rosbridge_suite and checkout `ros2` branch
  1. `colcon build`

## Output
```
--- stderr: rosbridge_msgs                                                 
CMake Error at /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find FastRTPS (missing: FastRTPS_INCLUDE_DIR FastRTPS_LIBRARIES)
Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
```

## Workaround
`CMAKE_PREFIX_PATH=$AMENT_PREFIX_PATH:$CMAKE_PREFIX_PATH colcon build`

There were some indications during debugging that rosbridge_suite might be using standard `make` instead of `ament_cmake`

--- a/rosbridge_msgs/package.xml
+++ b/rosbridge_msgs/package.xml
@@ -17,4 +17,7 @@
 
   <member_of_group>rosidl_interface_packages</member_of_group>
 
+  <export>
+    <build_type>ament_cmake</build_type>
+  </export>
 </package>

